features/step_definitions/description_change_steps.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require "faker"
+
+Given("a new planning application with a description of {string}") do |string|
+  @planning_application_with_description = FactoryBot.create(
+    :planning_application,
+    :not_started,
+    description: string,
+    local_authority: @officer.local_authority
+  )
+end
+
+Given("an existing description change request") do
```

Comment:
What's the difference with the user-simulated (hence, better) ` Given("I create a description change request from the application index with {string}") do |details|`?